imports/ui/pages/components/Modaler/GameListModal.js

```diff
@@ -54,63 +55,54 @@ const GameListModal = ({ gameList, isImported, history, onClose, ...rest }) => {
     return new Date(b.date) - new Date(a.date);
   });
 
-  const style = {
-    background: "#ffffff"
-  };
-
   return (
-    <Modal title={translate("myGames")} onCancel={onClose} footer={null} {...rest}>
-      <div style={style}>
+    <Modal title={translate("myGames")} width={1000} onCancel={onClose} footer={null} {...rest}>
+      <div style={{ background: "#ffffff" }}>
```

Comment:
Let's get rid of constant inline styles and put them into `./style.scss` file or CSS-in-JS approach. Check the https://www.sitepoint.com/react-components-styling-options/

Reply:
Ping me when you choose an approach you like so I can suggest how it can be implemented/integrated

Reply:
Let's also not forget that I want as much as possible to be maintainable via ICC administrators through the database. To that end, I was asking for CSS, as much as possible, for as much as makes sense, to be served to the client from the database and NOT hard coded, requiring a deploy to update.

Reply:
@djlogan2
Do you want to store all styles in database or some variables (like colors, margins, paddings, font-sizes)?
Should frontend code have default styles for cases when there is no styles in DB/request to DB failed/malformed styles in DB?

Reply:
Well, like I told you early on, where we draw the line is something we as the developers and architects have to figure out. The bottom line is that we do NOT have developers working for ICC, but we have many administrators. If we can reconfigure ICC via an administrative web page, we need write it thusly. If it's something that simply doesn't make sense to do, then we won't. For example, "themes", such as rounded corners, background colors, fonts, I think should be changable by an admin. Things like buttons and input boxes really can't be.  Right? Can we provide admins the ability to completely change any part or every part of the overall theme, or "look and feel" via the database?

Reply:
@djlogan2
Are you talking about WYSIWYG mode for administrators where they can change styling (like this one: http://angular-material.fusetheme.com/apps/dashboards/analytics - click gear button at the right)?

Regarding storing styles in DB I see two options:
1. Store theme variables such as rounded corners, background colors, fonts that will  be applied to styles on frontend - requires less effort but this approach provides less flexibility to administrators
2. Store ALL styles in DB as JSON (we plan to use `react-jss` library that utilizes JSON format for defining styles) - Need to think of interface for admins where they can change styling. This approach provides more flexibility to administrators but there may be a case when only CSS style might not be enough (layout changes also required) and we need WYSIWYG mode where admins can replace/resize/move elements via drag and drop and apply individual settings and styling. It requires much more  development effort than previous one.

Reply:
The "gear button" is one part of it, yes. "Maximum non-developer management", "Maximum non-developer maintenance", "Maximum non-developer administration" are thing we need to try to accomplish. If we can allow administrators to manage, maintain, and administer our application *without* a code change, we need to do that.